site/public/js/server.js

```diff
@@ -285,6 +290,41 @@ function newUploadCourseMaterialsForm() {
     $('#overwrite-materials-flag').remove();
 }
 
+function newUploadBanner() {
```

Comment:
This should be moved to an external script.

Reply:
Tried this and didn't go successful since depends on things from drag_and_drop, so decided it is a future thing